lib/zlib.js

```diff
@@ -613,9 +613,11 @@ function Zlib(opts, mode) {
          mode === UNZIP)) {
       windowBits = 0;
     } else {
+      // `{ windowBits: 8 }` is valid for deflate but nog gzip.
+      const min = Z_MIN_WINDOWBITS + (mode === GZIP);
```

Comment:
```suggestion
      const min = Z_MIN_WINDOWBITS + (mode === GZIP ? 1 : 0);
```

I think that would be a bit clearer here.